Essentials/src/main/resources/plugin.yml

```
@@ -505,8 +505,8 @@ commands:
     usage: /<command> <player>
     aliases: [etpaall]
   tpaccept:
-    description: Accepts a teleport request.
-    usage: /<command> [otherplayer]
+    description: Accepts most recent teleport request or one from a specific player or all tpahere requests.
```

Need not get so specific, this is what the usage strings and descriptions are for.
```suggestion
    description: Accepts teleport requests.
```